docs/guides/8to9guide.md

```diff
@@ -0,0 +1,94 @@
+---
+title: restify 8.x to 9.x migration guide
+permalink: /docs/8to9/
+---
+
+## Introduction
+
+restify `9.x` comes with `async/await` support!
+
+## Breaking Changes
+
+### Drops support for Node.js `8.x`
+
+Restify requires Node.js version `>=10.21.0`. 
```

Comment:
Is there anything in particular for 10.21.0 instead of just 10.x?

Reply:
I guess not... I will update it

Reply:
done